README.md

```diff
@@ -195,7 +195,7 @@ And now we can build from these inputs:
 $ cosa build
 ```
 
-Each build will write an OSTree commit into `$PWD/repo/` as well
+Each build will write an OSTree commit into `$PWD/tmp/repo/` as well
 as generate VM images in `$PWD/builds/`.
```

Comment:
Can probably just simplify this to:

> Each build will generate VM images in `$PWD/builds`.

The OSTree repo is no longer a separate output, but rather part of the `builds/` directory (https://github.com/coreos/coreos-assembler/pull/515). The `tmp/repo` one is just an implemetation detail.